torch/_inductor/scheduler.py

```diff
@@ -107,7 +107,7 @@ def debug_str(self) -> str:
         """Longer form printout for trace logs"""
         name = self.get_name()
         lines = [
-            f"{name}: {type(self).__name__}({type(self.node).__name__})",
+            f"{name}: {type(self).__name__}({type(getattr(self, 'node', None)).__name__})",
```

Comment:
Why did you need this? Shouldn't we go other way around and always define node?

Reply:
See thread on slack:
https://pytorch.slack.com/archives/C033H6DJSJU/p1699417275970679?thread_ts=1699412125.335719&cid=C033H6DJSJU

I wasn't able to repro the attribute error from slack here, but this should be a safe change.